{
    "id": "CVE-2020-27545",
    "product": "n/a",
    "version": "n/a ",
    "vulnerability": [
        "n/a"
    ],
    "description": "libdwarf before 20201017 has a one-byte out-of-bounds read because of an invalid pointer dereference via an invalid line table in a crafted object.",
    "poc": {
        "reference": [
            "https://www.prevanders.net/dwarfbug.html#DW202010-001"
        ],
        "github": [
            "No PoCs found on GitHub currently."
        ]
    }
}